scripts/packet/destroy-old-clusters.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+
+pushd "$(dirname "${BASH_SOURCE[0]}")"/../../test/cloudtest/pkg/providers/packet/packet_cleanup || exit 1
+go run ./... -t "$PACKET_AUTH_TOKEN" -p "$PACKET_PROJECT_ID" -k y -c y
```

Comment:
Let's modify packet_cleanup tool to use ENV variable instead of passing value via command line. Thanks,